README.md

```diff
@@ -1,4 +1,44 @@
 # bootstrap_go
 Lightweight bootstrap nodes for Golem network. Basic implementation in Go.
 
+# installation
+
+## prerequisites 
+- install `go` tools https://golang.org/doc/install  
+- create go workspace `mkdir ~/go`
+
+## get package and deps
+
+```
+go get github.com/golemfactory/bootstrap_go
+cd ~/go/src/github.com/golemfactory/bootstrap_go/
+# get dependencies and test dependencies
+go get -v -t ./...
+```
+
+## run
+```
+go run main/main.go 
+```
+
+## tests
+
+```
+go test -v -race ./...
+```
+where `-race` enables builtin race detector 
+
+## benchmarks
+
+```
+go test -bench=. -benchtime=20s ./...
+```
+
+# development
+
+## get code
+```
+git clone git@github.com:golemfactory/bootstrap_go.git
```

Comment:
`go get ...` is preferred way, since it's not gonna work if you clone the repo outside `$GOPATH`

Reply:
tests and benchmarks cloned outside works, but I'll conform to your comment

Reply:
Yeah, that's because you also did `go get` before that. Try removing the repo from your `GOPATH` and them running `go test` from the cloned repo.
Basically all imports in all files specify "github.com/golemfactory/bootstrap_go/..." so everything will be imported from your `GOPATH/src`, meaning that if you make a couple of changes in your custom repo they will not be considered when running.

Reply:
ok, thx for info